velox/docs/functions/binary.rst

```diff
@@ -34,6 +34,14 @@ Binary Functions
 
     Computes the HMAC with sha512 of ``binary`` with the given ``key``.
 
+.. function:: spooky_hash_v2_32(binary) -> varbinary
```

Comment:
Please use the same description as that in https://prestodb.io/docs/current/functions/binary.html